dali/operators/reader/loader/numpy_loader.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2017, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Reply:
those should all be fixed in the next push, I checked all the files I touched.